### ISSUE TITLE
key base completely not working today?

### ISSUE THREAD

**Issue by lreganCLDC** [keybase/keybase-issues#4226]
Does anyone have any information why key base seems to be entirely not working today?

**Comment by bleachbezel** [comment]
It's the same for me. I believed it was a part of #4219 but now I'm not so sure. #4219 was talking about api.keybase.io being a problem but this is different.

**Comment by vvdhsb** [comment]
Same. Unknown cert error on API reported in chats?

**Comment by jheasly** [comment]
Getting this error when I launch the desktop app:

> ERROR CODE 1601 - API network error: doRetry failed, attempts: 3, timeout 5.324s, last err: Get "https: / /api-0.core.keybaseapi.com/_/api/1.0/home. json?num_people=10 o&record_visit=0": ×509: certificate signed by unknown authority in method keybas
e. 1. home. homeGetScreen

**Comment by bleachbezel** [comment]
Reinstall the client on top of the installation! This will fix it!

**Comment by jheasly** [comment]
<img width="980" alt="Screenshot 2024-01-03 at 11 32 53 AM" src="https://github.com/keybase/keybase-issues/assets/96007/f1b6f3df-b296-4efb-8b84-7f567a2b3622">
Indeed it does! Thanks @bleachbezel!

**Comment by cidemaxio** [comment]
The Fedora/RedHat stable client still shows a certificate error (the latest macos works fine, today).

**Comment by mathieujobin** [comment]
@cidemaxio try downloading the latest from https://prerelease.keybase.io/

**Comment by sergei-zededa** [comment]
I also see the certificate error

**Comment by Vad133** [comment]
is it possible to install/work on windows 7? - after installation there is an error - entry point Discard VirtualMemory in kernel32 - so all windows 7 users should go to windows 10/11 due to this?)

**Comment by mathieujobin** [comment]
No, all Windows 7 users needs to upgrade because Microsoft stopped supporting it. 

![image](https://github.com/keybase/keybase-issues/assets/99191/75444f58-a5cd-4fe8-bb4c-b5d02b8e6343)
https://endoflife.date/windows

I prefer using Linux.

**Comment by rolandoalvarado** [comment]
Yah Keybase issue has been resolved by reinstalling client app.
